pkg/commands/server/capabilities.go

```diff
@@ -0,0 +1,59 @@
+package server
+
+import (
+	"github.com/urfave/cli"
+	"github.com/vapor-ware/synse-cli/pkg/client"
+	"github.com/vapor-ware/synse-cli/pkg/formatters"
+	"github.com/vapor-ware/synse-cli/pkg/utils"
+)
+
+const (
+	// capabilitiesCmdName is the name for the 'capabilities' command.
+	capabilitiesCmdName = "capabilities"
+
+	// capabilitiesCmdUsage is the usage text for the 'capabilities' command.
+	capabilitiesCmdUsage = "Enumerate all devices' capabilities of configured plugins on the active host"
+
+	// capabilitiesCmdDesc is the description for the 'capabilities' command.
+	capabilitiesCmdDesc = `The capabilities command hits the active Synse Server host's '/capabilities'
+  endpoint, which enumerates all devices' capabilities of configured plugins that are known to Synse
+  Server.
+
+Example:
+  synse server capabilities 
+
+Formatting:
+  The 'server capabilities' command supports the following formatting
+  options (via the CLI global --format flag):
+    - pretty (default)
+	- yaml
```

Comment:
minor, but this should be indented with spaces, for consistency

Reply:
Yeah, as I am looking through the descriptions across these codebase, there are some other places needed to be changed too.